### DISCUSSION TITLE
Cannot unlock the path to the 3rd biome

### DISCUSSION THREAD

**Discussion by LK24108** [FakeFishGames/Barotrauma#14769] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

i just a play with friends with this mods: multiplayer crew manager, dynamic europa, barocraftables, neurotrauma, lua, nt cybernetics, improved husk, network tweaks, betterhealthui, csforbarotrauma, movable and sellable wrecks, new wrecks for baro.
and on the border to 3-th location i has a problem, its doesn't skip me to 3-th location, i have a required reputation. but game says me talk with captain on station, even i talk with him its dont skip us.
sorry for my english i dont know this language very well.

### Reproduction steps

its just doesn't skip me

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

Would it be possible to share your save file here? That'd make it a lot easier for us to diagnose the issue. The saves are located in "C:/Users/[username]/AppData/Local/Daedalic Entertainment GmbH/Barotrauma". Note that you need to .zip the save first to attach it here, because github doesn't allow attaching files with the .save file type. Thank you and sorry for the inconvenience!

It's also worth noting that you can use the console command "togglecampaignteleport" to proceed: after you enter the command, you can teleport on the campaign map by double-clicking on a location. You will need to enable cheats (which disables achievements in the save) though, so it might not be what you want to do.

**Reply by LK24108** [reply]
thank you for command, i send you save, maybe you will find out what causes this error.
[baro save.zip](https://github.com/user-attachments/files/17253603/baro.save.zip)

**Comment by SomeRandomNoobKekeke** [comment]
I don't play with DE but i know there's some special "border events" https://discord.com/channels/103209684680323072/910489857028128818/1260263296414646323

**Reply by LK24108** [reply]
i too know it, but i have this ivent only on border 2-th location
on the border of 3-th location its change on outpost and this dont skip me on 3-th location

**Comment by ubertpendragon** [comment]
Dynamic Europa changes how biomes are unlocked, I don't know what you did but you broke it

**Reply by LK24108** [reply]
we dont do litteraly nothing to broke it, we just play, complete missions and buy a peoples.

**Reply by Regalis11** [reply]
It seems this is a bug in the Dynamic Europa mod. It overrides the "unlockpatheuropanridgeseparatists" event, but the event override doesn't unlock the path to the next biome.
